airflow/utils/log/log_reader.py

```diff
@@ -55,11 +55,14 @@ def read_log_chunks(
         contain information about the task log which can enable you read logs to the
         end.
         """
-        logs, metadatas = self.log_handler.read(ti, try_number, metadata=metadata)
+        kwargs = dict(log_type=log_type) if self.supports_triggerer else {}
```

Comment:
```suggestion
        kwargs = {"log_type": log_type} if self.supports_triggerer else {}
```

Same for other occurrences.